turingschool-examples/romcom
# Display saved covers

##What is the change? 
When a user clicks the view saved covers button the saved covers will be displayed together.

##Where should the reviewer start? 
Line 77

##How should this be tested?
 Click view saved covers button after having created covers and see if they are there.
